app/views/users/edit.html.haml

```diff
@@ -4,5 +4,6 @@
                   method: 'put',
                   wrapper: :horizontal_form,
                   html: { class: 'form-horizontal order-edit-form col-sm-6' }) do |f|
-  = render partial: 'form_fields', locals: { f: f }
+  = render partial: 'form_fields', locals: { f: f, 
+                                             from_sso: @user.directory == 'hydra_sso' }
```

Comment:
I think `User` should have a check method like `from_sso?
Also, you write a symbol to this field but check a string.